src/Jobs/Result/AbstractResult.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace iLUB\Plugins\TestCron\Jobs\Result;
+
+use ilCronJobResult;
+
+/**
+ * Class AbstractResult
+ *
+ * @package iLUB\Plugins\TestCron\Jobs\Result
+ * @author  Fabian Schmid <fs@studer-raimann.ch>
+ */
+abstract class AbstractResult extends ilCronJobResult {
```

Comment:
Not sure of those result abstractions are needed. maybe explain a little more here or remove.